src/map_bindings.cpp

```diff
@@ -42,4 +235,32 @@ PYBIND11_MODULE(map_strings, m)
         py::arg("output_file"),
         py::arg("write_idx") = 1,
         py::arg("threads") = 1);
-}
\ No newline at end of file
+
+  m.def("call_unitigs_existing", &py_uc_call_exists, "Calls unitigs and their colours in an existing Bifrost graph",
+          py::arg("graphfile"),
+          py::arg("coloursfile"),
+          py::arg("num_threads") = 1);
+
+  m.def("call_unitigs_build", &py_uc_call_build, "Builds and then calls unitigs in Bifrost graph",
+          py::arg("infile1"),
+          py::arg("kmer"),
+          py::arg("num_threads") = 1,
+          py::arg("is_ref") = 1,
+          py::arg("write_graph") = 0,
+          py::arg("infile2") = "NA");
+
+  m.def("query_unitigs_existing", &py_uc_query_exists, "Queries unitigs and their colours in an existing Bifrost graph",
+          py::arg("graphfile"),
+          py::arg("coloursfile"),
+          py::arg("query_file"),
+          py::arg("num_threads") = 1);
+
+  m.def("query_unitigs_build", &py_uc_query_build, "Builds and then queries unitigs in Bifrost graph",
+          py::arg("infile1"),
+          py::arg("kmer"),
+          py::arg("query_file"),
+          py::arg("num_threads") = 1,
+          py::arg("is_ref") = 1,
+          py::arg("write_graph") = 0,
+          py::arg("infile2") = "NA");
+}
```

Comment:
Can you add the version as an attribute:
```suggestion

    m.attr("version") = VERSION_INFO;
}
```
VERSION_INFO should be set up in setup.py

Reply:
Not sure how to implement this in setup.py. I can set version to be that in __init__.py however.